src/Client.php

```diff
@@ -396,9 +396,9 @@ private static function getVersion(): string
     {
         if (self::$version === null) {
             try {
-                self::$version = PrettyVersions::getVersion('mongodb/mongodb')->getPrettyVersion();
+                self::$version = InstalledVersions::getPrettyVersion('mongodb/mongodb') ?? 'unknown';
```

Comment:
Quoting the [Composer docs](https://getcomposer.org/doc/07-runtime.md#knowing-the-version-of-package-x):

> Returns the original version (e.g. v1.2.3) if vendor/package is installed, or null if it is provided/replaced, or throws OutOfBoundsException if the package is not installed at all